models/habits.js

```diff
@@ -5,27 +5,35 @@ const HabitsModel = (sequelize, DataTypes) => {
       primaryKey: true,
       unique: true,
       allowNull: false,
-      defaultValue: DataTypes.UUIDV4,
+      defaultValue: DataTypes.UUIDV4
     },
     name: {
       type: DataTypes.STRING,
       allowNull: false
     },
     userId: {
-      type: DataTypes.UUID,
+      type: DataTypes.UUID
+    },
+    expiresAt: {
+      type: DataTypes.STRING,
+      allowNull: true
```

Comment:
Just to clarify, this suggests that it is optional to provide an expiry date, yes?

Reply:
Oh, it shouldn't be, I will change it false.